services/apps/portal/src/alcs/document-grpc/alcs-document.message.interface.ts

```diff
@@ -0,0 +1,29 @@
+import { IsString } from 'class-validator';
+
+export class DocumentUploadRequestGrpc {
+  @IsString()
+  filePath: string;
+}
+
+export interface DocumentUploadResponseGrpc {
+  uploadUrl: string;
+  fileKey: string;
+}
+
+export class CreateDocumentRequestGrpc {
+  uploadedByUuid?: string;
+
+  mimeType: string;
```

Comment:
Does this also need @IsString()? Does that work here?

Reply:
We do not need it on ALCS side since it is only used for GRPC. In portal controller dto and grpc_dto usually identical so it is possible to use one model with validator tags

Reply:
Okay thanks for explaining